### PR TITLE
[READY] Update event partial homepage

### LINKED ISSUE #429: Make event section
Can we make an event/ workshop section on the landing page?

Header: 

**Bezoek ons hier:**

Text:
13 januari staat team LearningSpaces in Zwolle op hogeschool Windesheim tijdens het SAM evenement (link: http://samevent.nl). 
Can we do link on SAM  evenement?

25 januari staat team LearningSpaces op het Noorder eHealth-symposium (link: https://www.ehealthweek.net/event/noorder-ehealth-symposium/) in het UMCG. 
Can we dot the link on Noorderr eHealth-Symposium

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] add dutch event partial and rename it to event overview
- [x] add anchor for enabling linking to the specific partial 
- [x] remove past german event

Fixes #429 

## Screenshot
event-section

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
maybe update the kinda boring layout in another PR this week (to be decided).

**Comment by IwannaDJ** [comment]
Looking good! If you are going to make the pay out like this, can we change the tet? Update:

1st: Bezoek ons in Zwolle tijdens het Sam evenement op hogeschool Windesheim.

2nd: Tijdens het Noorder eHealth Symposium in het UMCG geven we twee workshops. Wil je aanwezig zijn? Neem contact (link to contact page?) met ons op.

Can we do something like a call to action in the events as well? Like the 2nd one?

File: source/localizable/events-overview/_events-overview.nl.md.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  </p>
+</div>
+
+<div class="agenda-item">
+  <h3>Noorder eHealth-symposium</h3>
+  <p class="info">25 januari 2017 | UMCG, Groningen</p>
+  <p>
+    Technologie vindt steeds beter zijn weg naar de patiënt. Tijdens het Noorder eHealth Symposium in het UMCG geven we twee workshops. We leggen je graag uit hoe we met <a href="http://www.learningspaces.nl" target="_blank">LearningSpaces</a> belangrijke informatie op een interactieve manier aan de patiënt leveren. Wil je aanwezig zijn? <a href="/contact/#form">Neem dan contact met ons op</a>.
+  </p>
+</div>
```
Comment (fatboypunk) [inline source/localizable/events-overview/_events-overview.nl.md.erb]: http://html5doctor.com/the-section-element/

Comment (RonaldDijkstra) [inline source/localizable/events-overview/_events-overview.nl.md.erb]: Yeah, thank you

**Pull request merged**